data/human/free worlds start.txt

```diff
@@ -1113,455 +1085,988 @@ mission "FW Pirates 2"
+			label next
+			`	"So, what do we do next?" you ask.`
+			`	"Well, the Terebinth is patched up, but they could use an escort back to <planet>. After that, you'll have to see what the Senate or the Council want you to do."`
 				accept
+	
+	npc accompany save
+		government "Free Worlds"
+		personality timid escort
+		ship "Falcon" "F.S. Terebinth"
+	
+	on visit
+		dialog phrase "generic arrived-without-npc dialog"
 
 
 
-mission "FW Pirates 3.2A"
+mission "FW Pirates: Diplomacy 4"
 	landing
-	name "Attack Greenrock"
-	description "Disobeying the Senate's orders, launch an all-out attack on the pirate world of <destination>, then land there."
-	source Trinket
-	destination Greenrock
-	clearance
+	name "Early Warning"
+	description "The early warning systems you installed on Hope have detected a heightened Navy presence. Travel to <destination> to defend against the potential Navy assault. Any punishment for the invasion of Greenrock will come later."
+	source "Longjump"
+	destination "Deep"
 	to offer
-		has "FW Pirates 3.2: done"
+		has "FW Pirates: Diplomacy 3: done"
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
 	
 	on offer
+		event "fw suppressed Greenrock"
 		conversation
-			`When you land on <origin>, you do indeed find a Free Worlds fleet gathered, but nowhere near as large a fleet as you had hoped for. As you survey the motley mix of grizzled militia veterans and young, naive interceptor pilots, you can't help but wonder if a fleet this size really has any chance at all against the strongest pirate world in southern space.`
-			`	Perhaps the Senate was right in suggesting a diplomatic solution. If you disobey their orders and lead this entire fleet to their deaths, that will not earn you any friends in the Free Worlds.`
+			`You arrive back on Longjump to a hero's welcome. People have heard of the battle you fought on Thule and the deal that was forged with the locals there. JJ and Freya greet you too, and congratulate you. But then Freya says, "We need to talk in private."`
+			`	You lead them into your ship, where Freya explains, "Soon after you left for Thule, Tomek took off on his own and raised a fleet for an attack on Greenrock. He thought the Senate's plan of making a deal with the pirate worlds was too cowardly, and that we needed a show of strength instead. Surprisingly, his stunt succeeded; Greenrock is now nominally under Free Worlds control."`
+			`	"This puts us in a rather difficult position," says JJ. "Much of Tomek's fleet didn't survive the assault, and those ships didn't come out of thin air, they were our ships intended for defensive purposes in case the Navy attacks again. Word of this has spread quickly. Thule is already second guessing their deal with us, and if Thule has heard of this then there's no doubt in my mind that the Republic has as well."`
```

Comment:
```suggestion
			`	"This puts us in a rather difficult position," says JJ. "Much of Tomek's fleet didn't survive the assault, and those ships didn't come out of thin air; they were intended for defensive purposes in case the Navy attacks again. Word of this has spread quickly. Thule is already second guessing their deal with us, and if Thule has heard of this then there's no doubt in my mind that the Republic has as well."`
```